### DISCUSSION TITLE
Reverse twitch chat

### DISCUSSION THREAD

**Discussion by james-walden** [Chatterino/chatterino2#5093] [Ideas]
Hi,

This has probably already been discussed before.

Is there anyway that you could make chatterino have a reverse option. where the newest twitch chat comes in at the top. There are other companies that already implement this, I prefer using chatterino, is there anyway to implement this?

Companies that already use it: BTTV and Restreamio

**Comment by pajlada** [comment]
You can find the previous discussion of this over here: https://github.com/Chatterino/chatterino2/issues/2821
The short answer is: While it would be cool, it requires quite a lot more work for us to do than for web-based technologies, so it's not on our roadmap right now.
